docs/argument-constraints.md

```diff
@@ -111,6 +111,20 @@ events in the context of an `A.CallTo`.
 So, tempting as it might be to save one of the constraints away in a
 handy variable, don't do it.
 
+## Using correct grammar
+
+FakeItEasy's API attempts to imitate the English language, so that call
+configurations and assertions read naturally. In that spirit, it's
+possible to use `An<T>` instead of `A<T>` for types whose name starts
+with a vowel. For instance:
+
+```csharp
+An<Apple>.That.Matches(a => a.Color == "Red")
+```
+
+`An<T>` is an exact synonym of `A<T>` and can be used exactly the same
+way.

```

Comment:
Maybe `A<T>` is a perfect synonym of `An<T>`. You're going to give `An<T>` a complex…
:grin:

Reply:
Indeed. Should I rephrase?

> `A<T>` and `An<T>` are exact synonyms, and can be used exactly the same way.

Reply:
I rephrased. I can roll it back if you don't like it.

Reply:
I like the new way. égalité !